.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: c
 sudo: required
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo /etc/init.d/postgresql stop; fi
```

Comment:
It is generally a better idea to run `service postgresql stop` instead of the script directly (see the `service` manpage). But OK for now :).